### PR TITLE
precompile heal effect

### PR DESCRIPTION
**Pull request by tcm390**
precompile heal effect material

Issue: (delay on the first effect)

https://user-images.githubusercontent.com/60634884/183993887-e529ae86-c43b-4f2d-ada5-8765890b17a0.mp4

Result:

https://user-images.githubusercontent.com/60634884/183994282-4321b817-ec80-41f5-9e2e-32aecf21d8b7.mp4

### CODE REVIEW

File: metaverse_modules/heal-effect/index.js [hunk 1]
```diff
@@ -229,13 +229,22 @@ export default () => {
...
+      app.add(group);
+      // pixelMaterial.onBeforeCompile = () => {
+      //   console.log('compile pixelMaterial')
+      // }
+      renderer.compile(group, camera)
 
-      let storeMaterial = false;
-      let particleAlreadyInScene= false;
+      let particleAlreadyInScene= true;
       let circlePlay = false;
```
Comment (avaer) [inline metaverse_modules/heal-effect/index.js:248]: If this is true, why do we need this variable?

**Review by avaer — CHANGES_REQUESTED** [review]
Code needs cleanup.

**Comment by alisaad673** [comment]
Seems to be working fine, there's no hiccup/delay/jitter now on the first time use of heal effect

**Comment by rondoor124** [comment]
@tcm390 would you do some code cleanup?

**Comment by tcm390** [comment]
> @tcm390 would you do some code cleanup?
[clean code](https://github.com/webaverse/app/pull/3581/commits/00da48541b54570644d38f8ee2a280d78624a0ca)

**Pull request closed without merging**